### PR TITLE
Joda Time for Android by dlew is used instead of the original.

### PR DESCRIPTION
**Pull request by tasomaniac**
They say that this is better in terms of performance.

### CODE REVIEW

File: app/proguard-rules.txt [hunk 1]
```diff
@@ -30,13 +30,9 @@
 # Volley
 -keep class android.support.v4.app.** { *; }
 -keep interface android.support.v4.app.** { *; }
--keep class com.actionbarsherlock.** { *; }
--keep interface com.actionbarsherlock.** { *; }
 -keep class com.android.volley.** { *; }
 -keep interface com.android.volley.** { *; }
 
--keepattributes *Annotation*
```
Comment (friedger) [inline app/proguard-rules.txt]: Why do you remove this?

Comment (tasomaniac) [inline app/proguard-rules.txt]: Because it is a duplicate in the configuration :)

**Comment by HenriqueRocha** [comment]
LGTM :+1:

**Comment by HenriqueRocha** [comment]
@tasomaniac This can't be merged. Can you rebase with develop and push again please?

**Comment by HenriqueRocha** [comment]
BTW, please add the library to `third_party.html` before we merge this

**Comment by tasomaniac** [comment]
oh ok. I will do both and then merge.

**Pull request merged**